src/Data/Fin/Properties.agda

```diff
@@ -674,6 +682,14 @@ combine-surjective {m} {n} i with remQuot {m} n i | P.inspect (remQuot {m} n) i
   (uncurry remQuot-combine)
   (combine-remQuot {m} n)
 
+private module ↔ {ℓ} = IsEquivalence (↔-isEquivalence {ℓ})
+
+v↔Fin : ∀ m n → Fin (m ^ n) ↔ Fin m ^v n
```

Comment:
Perhaps a slightly more meaningful name would be `Fin[m^n]↔Fin[m]^n`? Also could we import the `Vec` module qualified as `Vec` or something and then write `Fin m Vec.^ n` instead?

Reply:
I changed the name and wrote `Vec.^` instead of `^v`.